_docs/reference/release-notes.md

```diff
@@ -1,13 +1,84 @@
 ---
 title: Release Notes
-overview: What's been happening with Istio.
+overview: What's been happening with Istio
 
 order: 50
 
 layout: docs
 type: markdown
 ---
 
+## Istio 0.2
+
+#### General
+
+- **Updated Config Model**. Istio now uses the Kubernetes Custom Resource Definition model to describe and store its configuration. Configuration can now be 
+optionally managed using the `kubectl` command.
```

Comment:
one small nit that I forgot to mention yesterday: When running in Kubernetes environment, configuration can now be ...

Reply:
Done